src/funding-program.ts

```diff
@@ -20,4 +21,31 @@ export class FundingProgramApi {
   constructor(client: Marqeta, _options?: MarqetaOptions) {
     this.client = client
   }
+
+  /*
+   * Function to take the attributes of a new Program Funding source, create that
+   * in Marqeta, and return the Program Funding source information.
+   */
+  async create(funding: Partial<FundingProgram>): Promise<{
+    success: boolean,
+    funding?: FundingProgram,
+    error?: MarqetaError,
+  }> {
+    const resp = await this.client.fire('POST',
+      'fundingsources/program',
+      {},
+      { ...funding })
```

Comment:
Wouldn't the final paren be on the next line, in line with the `c` of `const`? Or are you wanting to change the style pattern?

Reply:
good catch